docs/UserGuide.md

```diff
@@ -299,11 +313,18 @@ Examples:
 Edits the existing room record at a specified index.
 
 Format: `oedit INDEX [r/ROOM_NUMBER] [t/ROOM_TYPE] [g/TAG]`
-* `INDEX` refers to the index number shown in the displayed room list. `INDEX` **must be a positive integer 1, 2, 3, …**.
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* Room occupancy status can only be changed through the `alloc` or `dealloc` command when a resident is allocated or deallocated. The occupancy status is not controllable through the `oedit` command. 
-* `oedit` will be blocked if the room is occupied. Run `dealloc` to deallocate the room before making further edits.
+* `oedit` will be blocked if the room is occupied. Run [`dealloc`](#deallocate-resident-from-room-dealloc) to deallocate the room before making further edits.
+* The occupancy status is not controllable through the `oedit` command.
+* Room occupancy status can only be changed through the `alloc` or `dealloc` command when a resident is allocated or deallocated. See [allocate a resident](#allocate-resident-to-room-alloc) or [deallocate a resident](#deallocate-resident-from-room--dealloc) for more info.   
+
```

Comment:
Is it necessary to bring in alloc ? Wdyt.

Reply:
I brought it in because an edit operation may be intended to change the allocation status. I think alloc and dealloc should both be mentioned in edit commands.

Reply:
Ok. I'll add it on my side.